docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,203 @@
+
+The `_airbyte_meta` field is an object that currently has one field:
+
+| Field Name     | Json Type | Description                                                                                |
+|----------------|-----------|--------------------------------------------------------------------------------------------|
+| `changes`      | list      | A record of any changes airbyte made to the data for compatibility and/or to handle errors |
+| `sync_id`      | integer   | Monotonically-increasing integer representing the current sync                             |
+
+The `changes` field is a list of objects, each of which represents a change to the data. Each object has the following fields:
+
+| Field Name | Json Type* | Description                                                                        |
+|------------|------------|------------------------------------------------------------------------------------|
+| `field`    | string     | The name of the field that was affected                                            |
+| `change`   | string     | The type of change (currently only `NULLED` or `TRUNCATED`)                        |
+| `reason`   | string     | The reason for the change, including its origin (source, platform, or destination) |
+
+These schemas are subject to change, however any change that is not backward compatible (ie, additive) will be accompanied by a breaking change notice.
+
+## Data Changes
+
+This version introduces changes to the data types used when writing to S3 in avro or parquet. The changes are as follows:
+
+### Primitive types
+
+All primitive types are unchanged:
+
+| Airbyte Json Schema Type  | Old Avro Type            | Old Parquet Type      | New Avro Type   | New Parquet Type |
+|---------------------------|--------------------------|-----------------------|-----------------|------------------|
+| string                    | `["null", "string"]`     | `Optional String`     | [UNCHANGED]     | [UNCHANGED]      |
+| boolean                   | `["null", "boolean"]`    | `Optional Boolean`    | [UNCHANGED]     | [UNCHANGED]      |
+| integer                   | `["null", "long"]`       | `Optional Int64`      | [UNCHANGED]     | [UNCHANGED]      |
+| number                    | `["null", "double"]`     | `Optional Double`     | [UNCHANGED]     | [UNCHANGED]      |
+
+### Date and time types
+
+This change introduces [this requested improvement](https://github.com/airbytehq/airbyte-internal-issues/issues/8973). Date and time types were stored in avro as unions of integral logical types and strings, which in parquet were represented as disjoint records. In practice, the date and time types were always converted, making the unions redundant. The resulting disjoint records were confusing.
```

Comment:
```suggestion
This change introduces [this requested improvement](https://github.com/airbytehq/airbyte-internal-issues/issues/8973). Date and time types were stored in Avro as unions of integral logical types and strings, which in Parquet were represented as disjoint records. In practice, the date and time types were always converted, making the unions redundant. The resulting disjoint records were confusing.
```

Reply:
can you say something other than "this requested improvement"; be more specific.

Reply:
`This change introduces [simplification of the handling of dates, times, and timestamps in Parquet and Avro]`?